robot_smach_states/src/robot_smach_states/util/designators/ed_designators.py

```diff
@@ -323,42 +323,36 @@ def _resolve(self):
         assert all(isinstance(v, FrameStamped) for v in vectors_of_interest)
 
         open_POIs = filter(self.is_poi_occupied, vectors_of_interest)
+        open_POIs.sort(key=self.distance_to_poi_area_heuristic)
```

Comment:
There is no frame_stamped given to distance_to_poi_area_heuristic. Does this work? In case it does please specify how!